js/modules/jericho.js

```diff
@@ -0,0 +1,151 @@
+export default {
+    state: {
+        profile: {
+            username: "Jericho",
```

Comment:
Please update your GitHUb username.

```suggestion
            username: "JerichoValladolid",
```

Reply:
Hi @JerichoValladolid, please resolve this in [jericho.js](https://github.com/JerichoValladolid/quizpocalypse/blob/master/js/modules/jericho.js).

Thanks!